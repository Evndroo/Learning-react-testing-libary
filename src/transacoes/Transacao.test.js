import React from 'react';
import Transacao from "./Transacao";
import { render } from "@testing-library/react"


describe("Componente de transação do extrato",()=>{

    it("Snapshot do componente deve ser sempre o mesmo", ()=>{
        const { container } = render(<Transacao data="18/11/2020" tipo="deposito" valor="100"/>);

        expect(container.firstChild).toMatchSnapshot();
    })

});