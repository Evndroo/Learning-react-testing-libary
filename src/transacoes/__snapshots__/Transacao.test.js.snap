// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componente de transação do extrato Snapshot do componente deve ser sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    18/11/2020
  </p>
  <p>
    deposito
  </p>
  <p>
    R$ 
    100
  </p>
</div>
`;
